spyder/plugins/variableexplorer/plotlib.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+"""
+Variable Explorer Plugin plotting library management.
+"""
+
+# Standard library imports
+import importlib
+
+# Local imports
+from spyder.config.base import _
+
+
+# Defining compatible plotting libraries
+# (default library is the first available, see `get_default_plotlib`)
+LIBRARIES = ("matplotlib", "guiqwt")
```

Comment:
I think these names should be capitalized to look good in our Preferences. What do you think?

Reply:
Both library logos display their names in all lowercase, so I would suggest not bothering with capitalizing them, hence simplifying code (key == value, as for the combo box management).